### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jsmi374**
#70

### CODE REVIEW

**Comment by nbol48** [comment]
Looks great

**Review by nbol48 — COMMENTED** [review]
Looks great.

**Review by pth6570 — COMMENTED** [review]
Looks good to me!

**Review by peterkobylarz — APPROVED** [review]
looks good!

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ Contributors:
 | Name | GitHub Username | Portfolio URL |
 | ---- | --------------- | ------------- |
 | Elsa Culler | eculler | https://eculler.github.io/ |
+| James Smith IV | jsmi374 | https://jsmi374.github.io/-jsmi374-.github.io/ |
```
Comment (earthlab-test-student) [inline README.md:9]: Looks good, I can open this link!

**Review by earthlab-test-student — APPROVED** [review]
Looks good!

**Pull request merged**